dandiapi/api/tasks/scheduled.py

```diff
@@ -63,6 +64,19 @@ def aggregate_assets_summary_task(version_id: int):
     version_aggregate_assets_summary(version)
 
 
+@shared_task(soft_time_limit=30)
+def recalculate_missing_sha256_checksums():
+    blobs_missing = AssetBlob.objects.filter(sha256__isnull=True).values_list('blob_id', flat=True)
```

Comment:
there should probably be some condition that blob creation time is 24 h ago or more since otherwise it might trigger on fresh blobs on which such a process already running or about to run (race condition)?